### PR TITLE
Update ES revisions, upgrade from Java 7 to 8.

### PR DESCRIPTION
**Pull request by UserNotFound**
Updates existing Elasticsearch versions to their latest revision.

The existing EOL Java 7 installer is no longer available, so this is as good of a time as any to upgrade to Java 8.

http://www.webupd8.org/2017/06/why-oracle-java-7-and-6-installers-no.html

### CODE REVIEW

File: 1.3/config.mk [hunk 1]
```diff
@@ -1,8 +1,7 @@
-# TODO - Upgrade to 1.3.9?
-export ES_VERSION = 1.3.2
-export ES_SHA1SUM = a7ea036468473084cd62e846f05a2069a987fb03
+export ES_VERSION = 1.3.9
+export ES_SHA1SUM = ee6013d33d6371184fae970ff8f1c2d76c0834a0
```
Comment (krallin) [inline 1.3/config.mk]: So, it looks like we don't have anyone using this image anymore (in fact we don't even make it available anymore) - we should probably just get rid of this one.

@fancyremarker, can you confirm  there are no exceptions here?

Comment (fancyremarker) [inline 1.3/config.mk]: There's only 1 exception: database ID 398 ... but it's on v1 and I'd be happy to upgrade it to 1.5 before migrating it to v2.

Comment (fancyremarker) [inline 1.3/config.mk]: To clarify further, [the plan](https://aptible.zendesk.com/agent/tickets/8106) was always to upgrade the database I mentioned to 1.5, so I do agree we should just remove 1.3 from this repo.

Comment (UserNotFound) [inline 1.3/config.mk]: Removed!

**Comment by krallin** [comment]
Thanks @UserNotFound - LGTM

**Pull request merged**